smart-freight-center/sfc-backend
# feat(auth): adds endpoint that generate token

## Description

This adds a `/auth/token` endpoint that allows you to generate tokens. 

### How to test it

1. Setup KeyCloak(see updated README in this branch)
2. Set all env variables
3. Run hit the `/auth/token` endpoint.



## Learning
- This implements the OpenID connect client_credentials flow
